specs/engage-digital_openapi3.yaml

```diff
@@ -370,39 +370,47 @@ paths:
         - Bots
   '/bots/handover':
     post:
-      description: This method is used to handover a thread from bot to agent or from agent to bot (only Messenger sources, see Bots API documentation)
+      description: This method is used to handover a conversation between a bot and an agent.
+        It handles 2 different cases, specific sources handover (Messenger for example, see Bots API doc)
+        and handover between Engage Virtual Agents and human agents
       operationId: handoverFromBotToAgent
       parameters:
-        - description: The foreign_id of the identity for which you want to do the handover.
+        - description: Who we are taking the conversation control from (either bot or agent).
           in: query
-          name: identity_foreign_id
+          name: from
           required: true
           schema:
             type: string
-        - description: The app that has thread control before handover.
+        - description: Who we are giving the convesation control to (either bot or agent).
           in: query
-          name: from
+          name: to
           required: true
           schema:
             type: string
-        - description: The app that should get thread control after handover.
+        - description: The type of handover (either type of channel or engage_virtual_agent).
           in: query
-          name: to
+          name: type
           required: true
           schema:
             type: string
-        - description: The type of channel.
+        - description: The foreign_id of the identity for which you want to do the handover (only for source specific handover).
           in: query
-          name: type
+          name: identity_foreign_id
           required: true

```

Comment:
I think we should remove the required here, as it is required only for type=messenger